JSDemos/Demos/Toolbar/Multiline/description.md

```diff
@@ -1,5 +1,5 @@
-## Configure Toolbar Items
+This demo demonstrates the multiline mode in the Toolbar component. To see step-by-step instructions on how to get started with the DevExtreme Toolbar component, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/). 
 
-## Specify Item Location
+In this demo, drag the handle of the component container on the right to resize the Toolbar. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, the Toolbar is displayed in multiple lines after resize. The number of lines is restricted by the minimum width of the container.
 
-## Customize Item Appearance
+In the single line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), if the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property of the item is set to `auto`, the item hides in the overflow menu with the width decrease.
```

Comment:
The sentence seems too long to me - I'm breaking it down into three shorter sentences. 

By the way, we don't use an article in phrases like "In XYZ mode".

```suggestion
In single-line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), the Toolbar doesn't wrap. It can display an overflow menu with items that didn't fit into the container. Use the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property to control whether an item appears in the overflow menu.
```